### ISSUE TITLE
NTR: Lipoprotein(a) low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8377]
New term request Lipoprotein(a) low in blood (elthat_140129053354): LPA; Lp(a)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000522
- label: Reduced circulating lipoprotein(a) concentration
- parents: [Hyperlipoproteinemia (HP:0010980)](https://hpo.jax.org/app/browse/term/HP:0010980)
- pmid: 
- def: The concentration of lipoprotein(a) in the blood circulation is below the lower limit of normal. Lipoprotein(a), also known as apolipoprotein(A), is encoded by LPA (HGNC:6667).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8377
- synonyms:
